site/docs/v1/tech/shared/_deleting-your-account.adoc

```diff
@@ -0,0 +1,20 @@
+To delete your account, take the following steps, in this order:
+
+1. delete all deployments
+2. switch your account to the community plan
+3. file a support ticket
+
+[NOTE.note]
+====
+FusionAuth require that customers delete their own deployments using the account portal.
+
+FusionAuth is **not able** to cancel accounts with deployments that could potentially contain critical user data.
+====
+
+To remove your deployments, please https://account.fusionauth.io/account/deployment/[log into your account and navigate to your deployments]. Once you see your deployment, you can click the [uielement]#Action# button and select [uielement]#Delete#. After a deployment is destroyed, you won't be charged for it.
+
+Then, https://account.fusionauth.io/account/plan/[switch your plan to to the community plan]. After the current billing cycle is done, you won't be charged. (Plans run for a month.)
+
+After deleting your deployments and changing your plan, your account will not be charged. It is inactive and can be deleted.
+
+After you have made your account inactive, please https://account.fusionauth.io/account/support/[open a support ticket to delete your account].
```

Comment:
What about the danger zone functionality that we released? This allows a user to remove their company.  They should not be opening a support ticket unless on invoice or fixed billing.


<img width="257" alt="image" src="https://github.com/FusionAuth/fusionauth-site/assets/16090626/77ca37d0-0e4d-406a-a4e9-dbba1e3cb289">